multi-node/aws/pkg/config/config.go

```diff
@@ -136,6 +136,8 @@ type Cluster struct {
 	KMSKeyARN                string            `yaml:"kmsKeyArn,omitempty"`
 	CreateRecordSet          bool              `yaml:"createRecordSet,omitempty"`
 	RecordSetTTL             int               `yaml:"recordSetTTL,omitempty"`
+	TLSCADurationDays        int               `yaml:"tlsCAValidDurationDays,omitempty"`
```

Comment:
nit: convention is that struct field names are exactly equivalent to yaml tags if you normalize capitalization. nothing explicitly relies on this convention, but i don't see a reason to break it.